--- evolution-ews-3.16.1/configure.ac.orig	2015-03-23 08:33:25.000000000 +0100
+++ evolution-ews-3.16.1/configure.ac	2015-04-18 19:47:58.913904347 +0200
@@ -199,9 +199,6 @@
 AC_SUBST(GETTEXT_PACKAGE)
 AC_DEFINE_UNQUOTED(GETTEXT_PACKAGE, "$GETTEXT_PACKAGE", [Package name for gettext])
 
-localedir='$(prefix)/$(DATADIRNAME)/locale'
-AC_SUBST(localedir)
-
 dnl ****************************
 dnl Initialize libtool
 dnl ****************************
